extension/js/common/platform/xss.ts

```diff
@@ -98,14 +98,14 @@ export class Xss {
           a.href = src;
           a.className = 'image_src_link';
           a.target = '_blank';
-          a.innerText = (title || 'show image') + (src.startsWith('data:image/') ? '' : ' (remote)');
+          a.innerText = (title || 'show image') + (src.startsWith('data:image/') || src.startsWith('cid:') ? '' : ' (remote)');
```

Comment:
Need to take a second look later

Reply:
No need, I reverted this addition, it's not needed after https://github.com/FlowCrypt/flowcrypt-browser/pull/3256/commits/f68b85f42d4064391a4664f2af2c0949777ab6b9